docs/components/concepts/incidents.md

```diff
@@ -6,15 +6,15 @@ description: "A process instance is stuck at a particular point, and requires us
 
 In Camunda Platform 8, an incident represents a problem in process execution. This means a process instance is stuck at a particular point, and requires user interaction to resolve the problem.
 
-Incidents are created in different situations, including the following:
+Incidents are created in the following scenarios:
 
-- A job is failed and it has no retries left.
-- An input or output variable mapping can't be applied.
-- A condition can't be evaluated.
-- A decision can't be evaluated.
+- Explicitly by job workers sending FAIL commands
+- Implicitly when a job is out of retries
+- Input/output-mapping cannot be applied
+- A condition or decision cannot be evaluated
```

Comment:
@akeller Sorry to go back on this but could we keep the original text here and in the versioned docs for 8.2? It was more precise, for example " Explicitly by job workers sending FAIL commands" (which I suggested) is not correct, it only creates an incident when no retries are left.

Reply:
I will adjust.